paasta_tools/api/views/instance.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+# Copyright 2015-2016 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+PaaSTA service instance status/start/stop etc.
+"""
+import json
+import logging
+import traceback
+
+from pyramid.response import Response
+from pyramid.view import view_config
+
+from paasta_tools import marathon_tools
+from paasta_tools.api import settings
+from paasta_tools.cli.cmds.status import get_actual_deployments
+from paasta_tools.marathon_serviceinit import compose_marathon_job_instance_count
+from paasta_tools.marathon_serviceinit import deploy_status_marathon_job
+from paasta_tools.marathon_serviceinit import get_bouncing_status
+from paasta_tools.utils import NoDockerImageError
+from paasta_tools.utils import PaastaColors
+from paasta_tools.utils import validate_service_instance
+
+log = logging.getLogger(__name__)
+
+
+def chronos_instance_status(instance_status, service, instance, verbose):
+    return
+
+
+def marathon_job_status(client, job_config):
+    mstatus = {}
+
+    try:
+        app_id = job_config.format_marathon_app_dict()['id']
+        mstatus['app_id'] = app_id
+    except NoDockerImageError:
+        error_msg = "Docker image is not in deployments.json. Has Jenkins deployed it?"
+        mstatus['message'] = error_msg
+        return mstatus
+
+    if marathon_tools.is_app_id_running(app_id, client):
+        app = client.get_app(app_id)
+        mstatus['deploy_status'] = PaastaColors.decolor(
+            deploy_status_marathon_job(app, app_id, client))
+
+        running_instances = app.tasks_running
+        mstatus['running_instances'] = running_instances
+        normal_instance_count = job_config.get_instances()
+        mstatus['normal_instance_count'] = normal_instance_count
+
+        status, instance_count = compose_marathon_job_instance_count(
+            running_instances, normal_instance_count)
+        mstatus['status'] = PaastaColors.decolor(status)
```

Comment:
Hm, I'm tempted to say I don't think we need this key. I think ``expected_instance_count`` and ``running_instances`` is enough - let the caller put them together and decide what's healthy/unhealthy.